docs/integrations/index.md

```diff
@@ -2,4 +2,5 @@
 title: "Integrations: Index"
 ---
 - [How To Write Integration (With Great Expectations) Documentation](../integrations/contributing_integration.md)
-- [Sample Integration](../integrations/integration_template.md)
\ No newline at end of file
+- [Sample Integration](../integrations/integration_template.md)
+- [DataHub Integration](../integrations/integration_datahub_template.md)
```

Comment:
```suggestion
- [DataHub Integration](../integrations/integration_datahub.md)
```

Hi!  I was just doing a review to make sure the table of contents updated properly for these and noticed that the path here wasn't quite right (probably just a typo left in from cut'n'paste).  I'm also going to suggest an additional change in the sidebars.js file so that this integration doc is displayed directly in the sidebar and not only as an item on the integrations index page.